app/bootstrap/cmd/main.go

```diff
@@ -51,73 +45,39 @@ func main() {
 		return
 	}
 
-	src := net.Source()
-	defer spiffe.CloseSource(src)
-	sv, err := src.GetX509SVID()
-	if err != nil {
-		log.FatalLn(fName,
-			"message", "Failed to get X.509 SVID",
-			"err", err.Error())
-		log.FatalLn(fName, "message", "Failed to acquire SVID")
-		return
-	}
-
-	if !svid.IsBootstrap(sv.ID.String()) {
-		log.Log().Error(
-			"Authenticate: You need a 'bootstrap' SPIFFE ID to use this command.",
-		)
-		log.FatalLn(fName, "message", "Command not authorized")
-		return
-	}
-
 	log.Log().Info(
 		fName, "FIPS 140.3 enabled", fips140.Enabled(),
 	)
 
+	// Panics if it cannot acquire the source.
+	src := net.AcquireSource()
+
 	log.Log().Info(
 		fName, "message", "Sending shards to SPIKE Keeper instances...",
 	)
 
+	api := spike.NewWithSource(src)
+	defer api.Close()
+
 	ctx := context.Background()
 
-	for keeperID, keeperAPIRoot := range env.Keepers() {
-		log.Log().Info(fName, "keeper ID", keeperID)
-
-		_, err := retry.Do(ctx, func() (bool, error) {
-			log.Log().Info(fName, "message", "retry:"+time.Now().String())
-
-			err := net.Post(
-				net.MTLSClient(src),
-				url.KeeperEndpoint(keeperAPIRoot),
-				net.Payload(
-					state.KeeperShare(
-						state.RootShares(), keeperID),
-					keeperID,
-				),
-				keeperID,
-			)
-			if err != nil {
-				log.Log().Warn(fName, "message", "Failed to send shard. Will retry.")
-				return false, err
-			}
-
-			log.Log().Info(fName, "message", "Shard sent successfully.")
-			return true, nil
-		},
-			retry.WithBackOffOptions(
-				retry.WithMaxInterval(60*time.Second), // TODO: to env vars.
-				retry.WithMaxElapsedTime(0),           // Retry forever.
-			),
-		)
+	// Broadcast shards to the SPIKE keepers until all shards are
+	// dispatched successfully.
+	net.BroadcastKeepers(ctx, api)
```

Comment:
This one, and the verification that follows it run forever...
We will need to have a "bail out" timeout as an env var; but that can be done as a follow-up PR.